src/source/Ice/IceAgent.c

```diff
@@ -89,7 +89,19 @@ STATUS createIceAgent(PCHAR username, PCHAR password, PIceAgentCallbacks pIceAge
                                             (PCHAR) pRtcConfiguration->iceServers[i].urls,
                                             (PCHAR) pRtcConfiguration->iceServers[i].username,
                                             (PCHAR) pRtcConfiguration->iceServers[i].credential))) {
+            pIceAgent->rtcIceServerDiagnostics[i].port = pIceAgent->iceServers[pIceAgent->iceServersCount].ipAddress.port;
```

Comment:
port is in network byte order. should convert it to machine byte order.

Reply:
Fixed.